src/controllers/countries.js

```
@@ -0,0 +1,32 @@
+const fetchCitiesByCountry = async (req, res) => {
+  const { countryCode } = req.params;
+
+  if (!countryCode) {
+    return res.status(400).json({ message: 'Country code is required' });
+  }
+
+  try {
+    const cities = await countriesService.getCitiesByCountryIso2(countryCode);
+    res.json(cities);
+  } catch (error) {
+    console.error(error);
```

ESLint

[error] Extra semicolon.

(semi)